src/dotnet/APIView/APIViewWeb/Pages/Assemblies/_CodeLine.cshtml

```diff
@@ -26,7 +26,14 @@
         <td colspan="2">
             @foreach (var lineDiagnostic in Model.Diagnostics)
             {
-                <p>
+                string diagClass = lineDiagnostic.Level switch
+                {
+                    APIView.CodeDiagnosticLevel.Error => "code-diagnostics-error",
+                    APIView.CodeDiagnosticLevel.Warning => "code-diagnostics-warn",
+                    APIView.CodeDiagnosticLevel.Info => "code-diagnostics-info",
+                    _ => "code-diagnostics-error"
+                };
+                <p class="@diagClass">
```

Comment:
why do we apply the class here and not to the same element as code-diagnostics

Reply:
Adding at a root row level will be an issue if there are multiple diag messages with different levels for a single line.